server/src/main/java/org/apache/cloudstack/network/router/deployment/VpcRouterDeploymentDefinition.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.cloud.network.router.deployment;
+package org.apache.cloudstack.network.router.deployment;
```

Comment:
Not blocking question: what's the reason for these package changes?

Reply:
@nvazquez 
in cloudstack codebase, old java classes are in package `com.cloud.xx.xx` ,  new java classes are in package `org.apache.cloudstack.xx.xx`

these classes are in packages `org.cloud.xx.xx` which are not correct I think. therefore move to `org.apache.cloudstack`